master/internal/checkpoint_gc.go

```diff
@@ -116,6 +117,14 @@ func (t *checkpointGCTask) Receive(ctx *actor.Context) error {
 		}, t.db, t.rm, t.taskLogger)
 
 		t.allocation, _ = ctx.ActorOf(t.allocationID, allocation)
+
+		t.Base.TaskContainerDefaults, err = t.rm.TaskContainerDefaults(
```

Comment:
can you leave a comment that `t.Base` is just a shallow copy of the `m.taskSpec` on the master, so you need to use caution when mutating it (though this case it fine, i think)

Reply:
Ack